### ISSUE TITLE
CRA Typescript error when using tw prop on html elements

### ISSUE THREAD

**Issue by reinvanimschoot** [ben-rogerson/twin.macro#33]
Hi guys

Just started experimenting with twin.macro and after having followed all instructions for CRA/Emotion, I'm getting this error when trying to convert my first `className` to `tw`:
```
Type '{ children: Element; tw: string; }' is not assignable to type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.
  Property 'tw' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'
```

I'm using CRA with the typescript template.

Is there anything else I need to adjust?

Cheers

**Comment by rbutera** [comment]
TypeScript support is pretty fresh. Only in the last alpha release.

Can you please confirm: The prop works as intended, it’s just typescript errors playing up right? 

We’ve got a PR in the works to update typescript support: see #24

**Comment by reinvanimschoot** [comment]
I can confirm it is typescript acting up.

**Comment by rbutera** [comment]
In the meantime you can suppress these errors by doing something like this:

instead of 

```typescript
const example = () => (<button type="submit" onClick={handleClick} tw="rounded p-4 bg-red-400">Click Me</button>)
```

do this

```
import tw from "twin.macro"

const Button = tw.button`rounded p-4 bg-red-400` // by convention people often use ButtonStyle instead

// below might be React.FunctionComponent ... I forgot

const example: React.ReactElement = ()=> (
 <Button type="submit" onClick={handleClick}> Click Me </Button>
)
```

basically you can use `tw.ELEMENT_NAME```

or using traditional styled-components syntax: `styled.button`${tw`bg-red-400`}` or `styled('button')`${tw`bg-red-400`}`

**Comment by ben-rogerson** [comment]
Typescript errors were fixed in twin@1.0.0-alpha.9
Closing this one up now 👍
